backend/config/locales/zu.yml

```diff
@@ -86,7 +86,7 @@ zu:
       defaults:
         categories: Select the categories that you work on
         mission: What's your mission?
-        mosaics: Select HeCo priority landscapes on which you will have impact
```

Comment:
this will have an impact on translations, right? I'm wondering if it might make sense to do it in an additive way, i.e. rather than renaming keeping 2 entries for a while, so that translations can be copied from the old one before it disappears (it's my understanding it disappears, but maybe I'm wrong about how transifex works)

Reply:
I also believe it will disappear, I can keep it for now, no problem ;)